airbyte-integrations/connectors/source-open-exchange-rates/source_open_exchange_rates/schemas/open_exchange_rates.json

```
@@ -0,0 +1,1040 @@
+{
+    "type": "object",
+    "required": [
+        "base",
+        "rates"
+    ],
+    "properties": {
+        "base": {
+            "type": "string"
+        },
+        "timestamp": {
+            "type": "string"
```

It seems like the returned value is an integer, not a string

```suggestion
            "type": "integer"
```